locales/notes.ca.properties

```diff
@@ -0,0 +1,80 @@
+# General note/notebook operations
+notebooks=Blocs de Notes
+tap-anywhere-to-start=Premeu a qualsevol lloc per iniciar
+note-info=Informacio de la nota
+add-to-contacts=Afegir a Contactes
+new-notebook=Crear Bloc de Notes
+notebook-all=Totes les notes
+notebook-trash=Paperera
+notebook-action-title=Editar Bloc de notes
+notebook-action-rename=Canviar el nom
+notebook-action-delete=Eliminar
+prompt-rename-notebook=Canviar el nom del Bloc de notes:
+prompt-delete-notebook=Premeu OK per eliminar aquest Bloc de notes. \n\nOBSERVACI&Oacute;: totes les vostres notes seran enviades a la paperera.
```

Comment:
Hello @dylano i received (about 24 hours ago) on my phone an update for the app Notes, and it contained the catalan translation !? Is it possible? i ask this because i don't see the catalan locale file on this repository ¿?

As you can see on my attached screenshot the word "Observació" appears perfectly! 

So, please, tell me if it's necessary that i make something else respect this question. 
Anyway, many many thanks for your consideration and help for include this locale into the stable version of the app ... asap... thanks!

![2016-01-15-16-59-31](https://cloud.githubusercontent.com/assets/1767547/12368590/117a421c-bbaa-11e5-99fa-5e0ca1c48f8f.png)